manager/volume.go

```diff
@@ -501,6 +499,61 @@ func (m *VolumeManager) Activate(volumeName string, frontend string) (v *longhor
 	return v, nil
 }
 
+func (m *VolumeManager) syncBackupAndRestoreWithLastBackup(volume *longhorn.Volume, engine *longhorn.Engine) (bool, error) {
+	// trigger BackupVolume synchronization
+	var backupVolumeName string
+	if volume.Status.RestoreRequired {
+		name, ok := volume.Labels[types.LonghornLabelBackupVolume]
+		if !ok {
+			return false, errors.New("Cannot find the backup volume label")
+		}
+		backupVolumeName = name
+	} else {
+		backupVolumeName = volume.Name
```

Comment:
When does the volume not have a label?

Reply:
I think it should be my bad. 

I originally used the name of the volume as the name of the backup volume when I judged that the restore is not needed, but it is not necessary. I will fix it to take the label value of the volume.